src/renderer/account/components/Management/Wallet/Wallet.js

```diff
@@ -37,10 +35,12 @@ export default class Wallet extends React.PureComponent {
   };
 
   render() {
-    const { className, encryptedMnemonic, secretWord, wallet, account } = this.props;
+    const { className, wallet, account } = this.props;
+    const { encryptedMnemonic, secretWord } = account;
 
     const coinType = COINS[wallet.coinType];
-    const identityColor = styles[coinType ? coinType.symbol.toLowerCase() : DEFAULT_COIN];
+    // TODO dont hardcode fallback color
```

Comment:
this is something we'll do when nOS has it's native coin. As we cannot include it yet in the coins consts for this reason